### PR TITLE
fix(intellij): add shortcut to focus task action

### PR DESCRIPTION
**Pull request by MaxKless**
there's now a shortcut in the gutter and the toolwindow
image
image

`ctrl shift g` because it's the same as `ctrl shift r` for running a target but `g` for graph.

There's something funky going on breaking shortcuts when you open the gutter actions with `option enter`... but that seems like a separate concern from this PR.

### CODE REVIEW

File: apps/intellij/src/main/kotlin/dev/nx/console/graph/actions/NxGraphFocusTaskAction.kt [hunk 1]
```diff
@@ -20,6 +18,10 @@ import kotlinx.coroutines.*
 class NxGraphFocusTaskAction(private val targetDescriptor: NxTargetDescriptor? = null) :
     DumbAwareAction() {
 
+    init {
+        shortcutSet = CustomShortcutSet.fromString("control shift G")
```
Comment (Cammisuli) [inline apps/intellij/src/main/kotlin/dev/nx/console/graph/actions/NxGraphFocusTaskAction.kt:22]: We should find a way to make this customizable via the keyboard map settings

**Review by Cammisuli — CHANGES_REQUESTED** [review]

File: apps/intellij/src/main/kotlin/dev/nx/console/graph/actions/NxGraphFocusTaskAction.kt (hunk 1 above)
Comment (MaxKless) [reply apps/intellij/src/main/kotlin/dev/nx/console/graph/actions/NxGraphFocusTaskAction.kt:22]: yeah fair enough. I can find some other predefined shortcut set. Any suggestions on the key combo?

Comment (Cammisuli) [reply apps/intellij/src/main/kotlin/dev/nx/console/graph/actions/NxGraphFocusTaskAction.kt:22]: You can keep the same combo, but just allow it to be configured with a custom keyboard shortcut.

Comment (MaxKless) [reply apps/intellij/src/main/kotlin/dev/nx/console/graph/actions/NxGraphFocusTaskAction.kt]: done

File: apps/intellij/src/main/kotlin/dev/nx/console/graph/ui/NxGraphBrowser.kt [hunk 2]
```diff
@@ -457,6 +457,7 @@ class NxGraphBrowser(
             val js =
                 """
             window.externalApi?.registerFileClickCallback?.((message) => {
+            console.log("file click", message)
```
Comment (Cammisuli) [inline apps/intellij/src/main/kotlin/dev/nx/console/graph/ui/NxGraphBrowser.kt:460]: Are these console.log's needed?

**Review by Cammisuli — APPROVED** [review]

**Pull request merged**